scripts/cicd/verify-multiaccount-records-have-been-ingested.py

```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/env -S python3 -u
+# Copyright 2024 Scalyr Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import time
+from http import HTTPStatus
+from typing import Dict, Set
+
+import requests
+from kubernetes import client, config
+from tabulate import tabulate
+
+
+def assert_env_non_empty(name):
+    if not os.environ.get(name):
+        print(f"ERROR: Environment variable {name} is not set or empty.")
+        sys.exit(1)
+
+
+ENV_TOKENS = [
+    "SCALYR_API_KEY_READ_TEAM_1",
+    "SCALYR_API_KEY_READ_TEAM_2",
+    "SCALYR_API_KEY_READ_TEAM_3",
+    "SCALYR_API_KEY_READ_TEAM_4",
+    "SCALYR_API_KEY_READ_TEAM_5",
+    "SCALYR_API_KEY_READ_TEAM_6",
+    "SCALYR_API_KEY_READ_TEAM_7"
+]
+
+ENV_ACCOUNT_NAME_MAPPING = {
+    "SCALYR_API_KEY_READ_TEAM_1": "ACCOUNT_NAME_1",
+    "SCALYR_API_KEY_READ_TEAM_2": "ACCOUNT_NAME_2",
+    "SCALYR_API_KEY_READ_TEAM_3": "ACCOUNT_NAME_3",
+    "SCALYR_API_KEY_READ_TEAM_4": "ACCOUNT_NAME_4",
+    "SCALYR_API_KEY_READ_TEAM_5": "ACCOUNT_NAME_5",
+    "SCALYR_API_KEY_READ_TEAM_6": "ACCOUNT_NAME_6",
+    "SCALYR_API_KEY_READ_TEAM_7": "ACCOUNT_NAME_7"
+}
+
+
+def validate_env():
+    for token in ENV_TOKENS:
+        assert_env_non_empty(token)
+
+    for account_name in ENV_ACCOUNT_NAME_MAPPING.values():
+        assert_env_non_empty(account_name)
+
+    assert_env_non_empty("SERVER_HOST")
+
+
+def query(token: str, server_host: str, time_start: str, retries: int):
+    def post_request():
+        return requests.post(
+            "https://app.scalyr.com/api/query",
```

Comment:
Should we also set `timeout` parameter for the requests method (e.g. 20 seconds or something reasonable)